features/admin.applications.v1/components/settings/sign-on-methods/step-based-flow/step-based-flow.tsx

```diff
@@ -946,6 +946,15 @@ export const StepBasedFlow: FunctionComponent<AuthenticationFlowPropsInterface>
         });
     };
 
+    console.log({
```

Comment:
Lets remove this

Reply:
Fixed: e125bf55f9488c283b429e504e45a4e63f7632b6